script.js

```diff
@@ -91,4 +91,26 @@ for (let i = 0; i < buttonsProject.length; i += 1) {
 popupExitButton.addEventListener('click', () => {
   popupWindow.style.display = 'none';
   backGround.style.display = 'none';
+});
+
+const contForm = document.getElementById('contact');
+const emailForm = document.getElementById('mail');
+const errorMessage = document.getElementById('error-message');
+
+const emailLowercase = (mail) => {
+  const itLowerc = new RegExp(/[A-Z]/g);
+  if (itLowerc.test(mail)) {
+    return false;
+  }
+  return true;
+};
```

Comment:
- [ ] Please you have to check on submitting the form if the email is all lowercase if not it should return a friendly message telling the user to input their email address all in lowercase.

Reply:
![image](https://user-images.githubusercontent.com/99298594/163284210-8c57ce7a-879d-4fb7-ba84-6918067923b8.png)
The image below shows the error message when you put some upper case to the email input, please clarify your feedback to improve my code.

Reply:
Great job done, everything Ok now